test/integration/enumerate_databases.test.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../src';
+
+describe('listDatabases', function () {
+  // TODO(NODE3860): Create driver test variants that run with AUTH enabled
```

Comment:
Am I reading this comment backwards?  It looks like these tests only run when auth is enabled.

Reply:
To add on to the question, which variants do these run under right now?

Reply:
currently using mlaunch's default auth which is username: 'user', password: 'password' (so to run that locally I add an --auth flag in the cluster_setup script for server and comment out the skip test)